iconsMapper.ts

```diff
@@ -8,15 +23,66 @@ const mdiViewBox = {
 
 const remap = reMapIcons();
 
+/**
+ * Get the icon lookup from the icon string
+ * @param icon
+ * @returns IconLookup
+ */
+function getIconName(icon: string): IconLookup {
```

Comment:
`IconLookup` is not defined

Reply:
is part of fontawesome package (don't remeber whitch).. ..your IDE should resolve it

Reply:
🙈 my bad... ..I had it imported.. ..probably killed due rebasing